iam-login-service/src/test/java/it/infn/mw/iam/test/X509Utils.java

```diff
@@ -0,0 +1,71 @@
+package it.infn.mw.iam.test;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class X509Utils {
+
+  public static final List<X509Cert> x509Certs = new ArrayList<X509Cert>();
+
+  static {
+
+    X509Cert x509Cert = new X509Cert();
+    x509Cert.display = "Personal Certificate";
+    x509Cert.certificate = new StringBuilder("-----BEGIN CERTIFICATE-----\n")
```

Comment:
is this class still needed and used? X509 account linking tests use the X509TestSupport class, and I remember having changed some other SCIM tests to use that as well.